main.py

```diff
@@ -32,9 +32,6 @@ def pagecountbynamespace( self ):
 	def pages_with_most_revisions( self ):
 		self.rep.pages_with_most_revisions()
 
-	def blank_pages( self ):
```

Comment:
I just see that this function do nothing - looks like the rest of this quarry [was removed](https://github.com/Niharika29/database-reports/commit/d0330efad1d5b0407b010b6629986c4e891ca429)